### PR TITLE
add cuda10.1 info

### PR DESCRIPTION
**Pull request by noorabani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,10 +76,12 @@ Each image is associated with one or more tags. Below is the list of images and
 - [IntelMPI GPU - cuda10 - Ubuntu 16.04](./base/gpu/intelmpi2018.3-cuda10.0-cudnn7-ubuntu16.04)
```
Comment (ninghu) [inline README.md]: missed here

**Comment by ninghu** [comment]
Please also add/update the docker files under base directory.

File: README.md (hunk 1 above)
Comment (noorabani) [reply README.md]: What is missed? The only cuda 10.1 image I have added is openmpi ubuntu 18.04

Comment (ninghu) [reply README.md]: [IntelMPI GPU - **cuda10.0** - Ubuntu 16.04]

Comment (noorabani) [reply README.md]: fixed

**Pull request merged**